Automattic/wp-calypso
# Remove Google+ share preview

#### Changes proposed in this Pull Request

* Removes the Google+ menu item from the sharing preview modal

#### Testing instructions



* Follow the instructions in https://github.com/Automattic/wp-calypso/issues/32105
* Verify that the expectation is fulfilled; the Google+ sharing preview is _not_ shown.

Fixes https://github.com/Automattic/wp-calypso/issues/32105

@Copons and/or @jhnstn Could I get your eyes here?

I believe this accomplishes the expectation in the associated issue, but I see references to Google+ in more places, like [here](https://github.com/Automattic/wp-calypso/blob/master/client/blocks/post-share/connection.jsx#L50), [here](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/marketing/connections/service-description.jsx#L53-L68), [here](https://github.com/Automattic/wp-calypso/blob/master/client/components/vertical-menu/items/social-item.jsx#L22), ... How deep should I be trying to scrub?

Also, I have 1 failing test on my branch. But the same test also fails for me on `master`. Technically that could mean I didn't introduce any new issues, but having failing tests on `master` seems fishy to begin with. Is this expected? Did I just happen to fork at an inopportune time?
<img width="762" alt="fail" src="https://user-images.githubusercontent.com/3772093/56553308-e114e280-6542-11e9-9e53-d2b96c86c317.png">
